IFC4x3/Properties/f/FaultDeformationStyle_1lYGGx_1jCCxEUZce$aPdE/Documentation.md

```diff
@@ -0,0 +1 @@
+Contain a human readable description of the style of deformation (e.g. brittle, ductile etc.) for the ShearDisplacementStructure. To report an identifier from a controlled vocabulary, deformationStyle_uri shall be used.
```

Comment:
```suggestion
Contain a human readable description of the style of deformation (e.g. brittle, ductile etc.) for the ShearDisplacementStructure. To report an identifier from a controlled vocabulary, deformationStyle_uri shall be used.

> NOTE: Definition according to OGC Geoscience Markup Language 4.1 (GeoSciML) (8.9.8.7 deformationStyle) 
> NOTE A controlled vocabulary may be found at  http://resource.geosciml.org/classifier/cgi/deformationstyle
```

Reply:
I would also remove "Contain" from the start of the description if you could amend your proposal @larswik